jenkins/performance-ci/nightly-performance-ci.groovy

```diff
@@ -61,26 +69,65 @@ pipeline {
         stage ("Deploy CORTX Cluster") {
             steps {
                 script { build_stage = env.STAGE_NAME }
+
                 script {
-                    def cortxCluster = build job: '/Cortx-Automation/RGW/setup-cortx-rgw-cluster', wait: true,
-                    parameters: [
-                        string(name: 'CORTX_RE_BRANCH', value: "${CORTX_RE_BRANCH}"),
-                        string(name: 'CORTX_RE_REPO', value: "${CORTX_RE_REPO}"),
-                        string(name: 'CORTX_SERVER_IMAGE', value: "${CORTX_SERVER_IMAGE}"),
-                        string(name: 'CORTX_DATA_IMAGE', value: "${CORTX_DATA_IMAGE}"),
-                        string(name: 'CORTX_CONTROL_IMAGE', value: "${CORTX_CONTROL_IMAGE}"),
-                        string(name: 'DEPLOYMENT_METHOD', value: "standard"),
-                        text(name: 'hosts', value: "${hosts}"),
-                        string(name: 'EXTERNAL_EXPOSURE_SERVICE', value: "NodePort"),
-                        string(name: 'SNS_CONFIG', value: "${SNS_CONFIG}"),
-                        string(name: 'DIX_CONFIG', value: "${DIX_CONFIG}")
-                    ]
-                    env.cortxcluster_build_url = cortxCluster.absoluteUrl
-                    env.cortxCluster_status = cortxCluster.currentResult
-                }
-            }
+                    if (params.infrastructure == "HW") {
+                        sh label: 'Copy RPMS', script:'''
+                        echo "Deploying CORTX on Hardware"
+                        '''
+                        checkout([$class: 'GitSCM', branches: [[name: '$CORTX_RE_BRANCH']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'AuthorInChangelog']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: '$CORTX_RE_REPO']]])
+
+                        sh label: 'Copy RPMS', script:'''
+                        pushd solutions/kubernetes/
+                            curl -l https://raw.githubusercontent.com/shailesh-vaidya/cortx-re/hw-performance-ci/jenkins/performance-ci/nightly-performance-ci.yml -o solution.yaml
```

Comment:
please change repository to main repository if test is completed

Reply:
fixed